sources/web/datalab/polymer/modules/Utils.ts

```diff
@@ -102,6 +102,26 @@ class Utils {
   }
 
   /**
+   * Parses the given querystring and returns a dictionary of keys and values.
+   * @param search querystring part of the url to parse, typically window.location.search
+   */
+  static getSearchParams(search: string) {
```

Comment:
Why not new URLSearchParams? This isn't something that I would recommend doing yourself.